### PR TITLE
Remove unnecessary space on link

### PR DESCRIPTION
**Pull request by lunny**
The action run title has a blank. This PR removes it.

### CODE REVIEW

File: templates/repo/actions/runs_list.tmpl [hunk 1]
```diff
@@ -7,7 +7,7 @@
 			<div class="issue-item-main gt-f1 gt-fc gt-df">
 				<div class="issue-item-top-row">
 					<a class="index gt-ml-0 gt-mr-2" href="{{if .Link}}{{.Link}}{{else}}{{$.Link}}/{{.Index}}{{end}}">
-						{{.Title}}
+						{{- .Title -}}
```
Comment (delvh) [inline templates/repo/actions/runs_list.tmpl:10]: What's exactly the meaning of this `- <something> -`?
Does it remove trailing spaces starting from `>` until the `<`, or what?

Comment (lunny) [reply templates/repo/actions/runs_list.tmpl:10]: > What's exactly the meaning of this `- <something> -`? Does it remove trailing spaces starting from `>` until the `<`, or what?

Yes. ref https://pkg.go.dev/text/template#hdr-Text_and_spaces

**Review by delvh — APPROVED** [review]

**Comment by delvh** [comment]
Good to know.
I've already wondered how to get rid of these annoying spaces between some things.
Seems like you just gave me an idea for future PRs…

**Review by jolheiser — APPROVED** [review]

**Review by yardenshoham — APPROVED** [review]

**Comment by jolheiser** [comment]
🎺 🤖

**Pull request merged**